build.gradle

```diff
@@ -26,11 +26,13 @@ repositories {
 dependencies {
     def versions = [
         'agp': '7.0.4',
-        'spock': '2.2-M1-groovy-3.0'
+        'sdkBuildTools': '30.0.4',
```

Comment:
why was this added?

Reply:
`com.android.builder.model.Version` is deprecated in favor of `com.android.Version`, which seems to be located in `com.android.tools:common` as part of the sdk build tools.

Reply:
this should be automatically brought in as a transitive dependency, is it not?

Reply:
It's not, according this [build scan](https://ge.solutions-team.gradle.com/s/tpfi3htecs3uk/dependencies?expandAll&toggled=W1swLDEsWzddXSxbMCwxLFs3LDhdXSxbMCwxLFs3LDEwXV0sWzAsMSxbMF1dXQ).

Reply:
This is interesting, we want to make sure we are using the version bundled in with AGP and not the version selected by the cache fix plugin. `compileOnly` should achieve this.

Reply:
Thanks for checking.

Reply:
@runningcode What was the outcome of this thread?

Reply:
I created a sample Android project with a task like this that included this cache fix plugin (but not, explicitly, the SDK build tools):

```tasks.register('printAndroidVersion') {
    def agpVersion = Version.ANDROID_GRADLE_PLUGIN_VERSION
    def androidToolsBaseVersion = Version.ANDROID_TOOLS_BASE_VERSION

    inputs.property('agpVersion', agpVersion)
    inputs.property('androidToolsBaseVersion', androidToolsBaseVersion)

    doLast {
        println "Android Gradle Plugin Version: $agpVersion"
        println "Android Tools Base Version: $androidToolsBaseVersion"
    }
}
```

Changing the AGP version showed that sample project used the version of SDK Build Tools bundled with AGP, the desired behavior.

Reply:
Thanks for the details, @clayburn